data/json/itemgroups/food_service.json

```diff
@@ -1800,6 +1801,14 @@
     "container-item": "bag_paper_powder",
     "entries": [ { "item": "sugar", "container-item": "null", "count": 71 } ]
   },
+  {
+    "type": "item_group",
+    "id": "sugar_bag_paper_50lb",
+    "subtype": "collection",
+    "//": "This group was created by a fool and may contain errors.",
```

Comment:
Can we not do those? It's not the first time people see the automated comment and add this exact comment to their itemgroups. I really don't think we need to propagate this any further.

Reply:
Well it's not automated at least. :) but sure if you'd prefer I remove my change to it I'll do so once I've actually gotten this right.  As of this morning it was not right and containing errors.